.github/workflows/wheels.yml

```diff
@@ -215,7 +215,7 @@ jobs:
           SKLEARN_SKIP_NETWORK_TESTS: 1
 
       - name: Store artifacts
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           path: dist/*.tar.gz
```

Comment:
I'd recommend naming the artifact here; using the default name is tricky now since you can only do it one time.